core/dbt/logger.py

```diff
@@ -15,17 +15,16 @@
 import logbook
 from dbt.dataclass_schema import dbtClassMixin
 
-# Colorama needs some help on windows because we're using logger.info
-# intead of print(). If the Windows env doesn't have a TERM var set,
-# then we should override the logging stream to use the colorama
-# converter. If the TERM var is set (as with Git Bash), then it's safe
-# to send escape characters and no log handler injection is needed.
-logging_stdout = sys.stdout
-if sys.platform == "win32":
-    if not os.getenv("TERM"):
-        logging_stdout = colorama.AnsiToWin32(sys.stdout).stream
```

Comment:
`colorama.AnsiToWin32(sys.stdout).stream` is only used when `wrap=False`. When changing to hit this for Windows only terminals, we can set `wrap=True` and drop the `AnsiToWin32` call